backend/.gitignore

```diff
@@ -0,0 +1,2 @@
+#backend
+/venv
```

Comment:
Why are you on windows?

Reply:
Anyway, please add backward slash.

Reply:
no I'm using linux

Reply:
Wait. Sorry. I got confused. Never mind.

Reply:
what shd i add backslash for?